sources/web/datalab/polymer/components/item-list/item-list.ts

```diff
@@ -12,6 +12,19 @@
  * the License.
  */
 
+type ColumnTypeList = Date|number|string;
```

Comment:
This type name sounds to me like it is a list of the ColumnType types defined just below. How about something like "ColumnJavascriptType"?

Reply:
That's probably OK.

Reply:
Done.